### PR TITLE
Revert active-response test changes

### LINKED ISSUE #5724: Revert test_api_endpoint_performance expected fails logic
## Description
During testing of the 4.9.0 release, a critical regression was identified in the Workload benchmarks metrics tests. After investigating the issue in [Research possible break change in Workload benchmarks metrics tests](https://github.com/wazuh/wazuh-jenkins/issues/6920), it was determined that the regression was caused by changes introduced in the test_api_endpoint_performance test, as part of https://github.com/wazuh/wazuh-qa/pull/4657.

It is necessary to revert the changes in test_api_endpoint_performance to align with standardized test procedures.

## Tasks

- [x] Revert https://github.com/wazuh/wazuh-jenkins/issues/6920
- [x] Analyze the [original issue](https://github.com/wazuh/wazuh-qa/issues/3665) and determine if the suggested changes need to be implemented differently. Develop a coherent plan to test the expected failing tests.

## Validation
- [x] Workload benchmarks metrics tests pass and results match with 4.8.x

### PR DESCRIPTION
**Pull request by GGP1**
# Description

Reverts the changes from https://github.com/wazuh/wazuh-qa/pull/4657.

## Testing performed

|OS|Package used|
|--|--|
| Debian 10 | 4.9.0-1 |

| Validation | Jenkins | Local  | OS  | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|
|    [Build](https://ci.wazuh.info/job/CLUSTER-Workload_benchmarks_metrics/707)       | 🟢 ⚫ | ⚫⚫ |         |         | Nothing to highlight |

### CODE REVIEW

**Review by javiersanchz — DISMISSED** [review]
LGTM !

**Comment by nico-stefani** [comment]
LGTM!

**Review by nico-stefani — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -102,7 +102,6 @@ s
 - Added tests for checking agent status upon ungraceful closure.([#4146](https://github.com/wazuh/wazuh-qa/pull/4146)) \- (Tests)
 - Agent syncronization testing after group deleting ([#4143](https://github.com/wazuh/wazuh-qa/pull/4143)) \- (Tests)
 - Add test for AWS Custom Logs. ([#4675](https://github.com/wazuh/wazuh-qa/pull/4675)) \- (Tests)
-- Add new behaviour for endpoints marked as xfail in api_endpoints_performance test ([#4657](https://github.com/wazuh/wazuh-qa/pull/4657)) \ (Tests)
```
Comment (juliamagan) [inline CHANGELOG.md]: We should add a `Deleted` entry in the 4.9.1 changelog, not delete it from 4.8.0, because in 4.8.0 the change is still there.

**Review by juliamagan — CHANGES_REQUESTED** [review]
GJ! Just a minor change.

**Review by juliamagan — APPROVED** [review]

**Pull request merged**